roles/kubernetes-apps/ansible/tasks/netchecker.yml

```diff
@@ -23,6 +23,7 @@
 - name: Kubernetes Apps | Netchecker Templates list
   set_fact:
     netchecker_templates:
+      - {file: netchecker-ns.yml, type: ns, name: netchecker-namespace}
```

Comment:
Will this fail if the namespace is set to `kube-system` or when the namespace already exists?

Reply:
It will work in both those cases.

kubectl will basically apply an upsert to the namespace, so I suppose this could override the namespace labels if a cluster had some other custom labels defined for an existing namespace.  

So better approach might be to check that the namespace does not exist before applying the namespace manifest.  What are your thoughts?

Reply:
I think we're good :)